pincer/objects/app/interactions.py

```diff
@@ -145,33 +145,36 @@ def __post_init__(self):
             return
 
         for option in self.data.options:
-            if option.type is AppCommandOptionType.STRING:
+            if option.type == AppCommandOptionType.STRING:
```

Comment:
@Lunarmagpie not sure, but this is alright?

Reply:
#425 the "is" statement is not working for me in python 3.9

Reply:
Okay, I'm assuming that our APIObject isn't properly converting the socket payload to an object. As I'm not up to date with the current state of the way the lib handles this, I can't provide a solution rn.  
  
One thing I do know is that this is not the expected behavior, and we should model around the expected behavior _(which is that there is an object, so a `is`_ statement should be used_). Nonetheless,_ this issue should be addressed.